addons/mail/static/src/models/channel.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one, many } from '@mail/model/model_field';
+import { insertAndReplace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Channel',
+    identifyingFields: ['id'],
+    recordMethods: {
+        async fetchChannelMembers() {
+            const channelMembersList = await this.messaging.rpc({
+                model: 'mail.channel',
+                method: 'load_more_members',
+                args: [[this.id]],
+                kwargs: {
+                    known_member_ids: this.channelMembers.map(channelMember => channelMember.partner.id),
+                },
+            });
+            if (!this.exists()) {
+                return;
+            }
+            this.update({
+                memberCount: channelMembersList.count,
+                channelMembers: channelMembersList.channelMembers,
```

Comment:
rpc should return update data directly, not a strange dict that has to be reformatted

`this.update(channelData)`